_docs/graphql-api.md

```diff
@@ -0,0 +1,399 @@
+---
+title: GraphQL Data API
```

Comment:
I want to include "Insights" in title as an easy name. Could we make the title "Insights GraphQL API"?